crates/api/src/consts.rs

```diff
@@ -1,2 +1,5 @@
 pub const SYSTEM_PROMPT_MAX_LEN: usize = 64 * 1024;
 pub const LIMIT_MAX: i64 = 100;
+
+/// Allowed proxy paths that can be forwarded to OpenAI
+pub const ALLOWED_PROXY_PATHS: &[&str] = &["responses", "chat/completions", "models"];
```

Comment:
Let's carefully review the endpoints that we'd like to whitelist. I think we're missing below endpoints:

- `/model`: e.g. `/model/list`
- `/conversations`
- `/files`
- `/signature`